src/graphql/syncTemplates.ts

```diff
@@ -67,12 +67,7 @@ export async function syncTemplate(
   );
 
   // If Template exists, update it, else create new
-  if (existingTemplate) {
```

Comment:
saves a bunch of network requests

Reply:
@michaelfarrell76 can you update the comment? Basically this is just create-if-not-exists rather than update-or-create